com.unity.render-pipelines.universal/Editor/ShaderGUI/ShadingModels/LitGUI.cs

```diff
@@ -72,6 +85,11 @@ public struct LitProperties
             public MaterialProperty highlights;
             public MaterialProperty reflections;
 
+            public MaterialProperty clearCoat;  // Enable/Disable dummy property
+            public MaterialProperty clearCoatMap;
+            public MaterialProperty clearCoatMask;
```

Comment:
I must admit that reading these two together made me confused. 
Map here is a MaskMap (rg: mask, smoothness) right?

Reply:
Yes. And now only r:mask.